src/api.ts

```diff
@@ -1,7 +1,7 @@
 import { MovieData } from './types/MovieData';
 import { ResponseError } from './types/ReponseError';
 
-const API_URL = 'https://www.omdbapi.com/?apikey=your-key';
+const API_URL = 'https://www.omdbapi.com/?apikey=a570192';
```

Comment:
it's ok use api key like here in test projects but anyway, read this [article](https://create-react-app.dev/docs/adding-custom-environment-variables/) JFYI